### PR TITLE
Car docs: simplify and sort by experience tiers

### PR DESCRIPTION
**Pull request by sshane**
- Cars that need a pedal to gain full-range ACC are not "openpilot Longitudinal" cars
- Cars that don't have steering at any speed above 0 mph are not "Steering to 0mph" cars
- Assumed all Toyota, Lexus, Volkswagen, and Hyundai (that have [max torque set to 384](https://github.com/commaai/openpilot/blob/master/selfdrive/car/hyundai/values.py#L18)) to have "Good lateral torque"
- Alphabetical sorting
- All cars previously outside of the community list are "Well supported." Only the cars in the old community supported list that have harnesses are now well supported as well.
- Removed Well Supported from Genesis G90, moved to dashcam. No routes in last year

Todos:
- [x] VW well supported?
- [x] ~All Hyundai well supported?~ Used whether https://comma.ai/vehicles said the vehicle was supported and had a harness. Feel free to review
- [x] Do all VW have good torque?
- [x] ~Add exception back for pedal cars~

### CODE REVIEW

**Comment by geohot** [comment]
Is the sorting done? Why is "RAV4 Hybrid 2016-18" not gold, and why is "Civic Coupe 2016-18" not bronze?

**Comment by geohot** [comment]
Replace "Lateral to 0mph" with "Steering to 0mph"

File: docs/CARS.md [hunk 1]
```diff
@@ -1,185 +1,200 @@
...
-| Volkswagen| Tiguan 2020                     | Driver Assistance | Stock            | 0mph               | 0mph         |
-| Volkswagen| Touran 2017                     | Driver Assistance | Stock            | 0mph               | 0mph         |
+## Gold Cars
 
-<sup>1</sup>Requires an [OBD-II car harness](https://comma.ai/shop/products/comma-car-harness) and [community built ASCM harness](https://github.com/commaai/openpilot/wiki/GM#hardware). ***NOTE: disconnecting the ASCM disables Automatic Emergency Braking (AEB).*** <br />
-<sup>2</sup>Not including the China market Kamiq, which is based on the (currently) unsupported PQ34 platform. <br />
-<sup>3</sup>Not including the USA/China market Passat, which is based on the (currently) unsupported PQ35/NMS platform. <br />
-<sup>4</sup>Model-years 2021 and beyond may have a new camera harness design, which isn't yet available from the comma store. Before ordering,
+| Make    | Model (US Market Reference) | Supported Package | openpilot Longitudinal | Longitudinal to 0mph | Steering to 0mph | Good steering torque | Well Supported |
```
Comment (sshane) [inline docs/CARS.md:16]: `ACC from 0mph`?

File: docs/CARS.md [hunk 2]
```diff
@@ -1,185 +1,200 @@
...
+| Audi       | Q2 2018                         | ACC + Lane Assist |                        |          ⭐           |        ⭐         |          ⭐           |       ⭐        |
+| Audi       | Q3 2020-21                      | ACC + Lane Assist |                        |          ⭐           |        ⭐         |          ⭐           |       ⭐        |
+| Audi       | S3 2015                         | ACC + Lane Assist |                        |          ⭐           |        ⭐         |          ⭐           |       ⭐        |
+| Genesis    | G70 2018                        |        All        |                        |          ⭐           |        ⭐         |          ⭐           |       ⭐        |
+| Genesis    | G70 2020                        |        All        |     ⭐<sup>3</sup>      |          ⭐           |        ⭐         |          ⭐           |                |
+| Genesis    | G90 2018                        |        All        |     ⭐<sup>3</sup>      |          ⭐           |        ⭐         |                      |       ⭐        |
```
Comment (adeebshihadeh) [inline docs/CARS.md:75]: I think all Hyundai should have good torque

Comment (adeebshihadeh) [inline docs/CARS.md]: I think all HKG should have good torque

File: docs/CARS.md [hunk 3]
```diff
@@ -1,185 +1,200 @@
...
+| Hyundai    | Ioniq Electric 2020             |    SCC + LKAS     |                        |          ⭐           |        ⭐         |          ⭐           |       ⭐        |
```
Comment (adeebshihadeh) [inline docs/CARS.md:76]: <s>All HKG should have long too</s>, only counting those with radar points for now

File: docs/CARS.md [hunk 4]
```diff
@@ -1,185 +1,200 @@
...
+| Chrysler   | Pacifica 2017-18              |  Adaptive Cruise  |                        |          ⭐           |                  |                      |                |
+| Chrysler   | Pacifica 2020                 |  Adaptive Cruise  |                        |          ⭐           |                  |                      |                |
+| Chrysler   | Pacifica Hybrid 2017-18       |  Adaptive Cruise  |                        |          ⭐           |                  |                      |                |
+| Chrysler   | Pacifica Hybrid 2019-21       |  Adaptive Cruise  |                        |          ⭐           |                  |                      |                |
+| GMC        | Acadia 2018<sup>4</sup>       |  Adaptive Cruise  |           ⭐            |          ⭐           |                  |                      |                |
+| Genesis    | G80 2018                      |        All        |                        |          ⭐           |        ⭐         |                      |       ⭐        |
```
Comment (adeebshihadeh) [inline docs/CARS.md]: has good torque

File: docs/CARS.md [hunk 5]
```diff
@@ -1,185 +1,200 @@
...
+| Hyundai    | Elantra 2017-19               |    SCC + LKAS     |                        |                      |                  |                      |                |
+| Hyundai    | Genesis 2015-16               |    SCC + LKAS     |                        |                      |                  |                      |       ⭐        |
+| Hyundai    | Ioniq Electric 2019           |    SCC + LKAS     |                        |          ⭐           |                  |                      |       ⭐        |
+| Hyundai    | Ioniq Hybrid 2017-19          |    SCC + LKAS     |                        |          ⭐           |                  |                      |                |
+| Hyundai    | Ioniq Hybrid 2020-22          |     SCC + LFA     |                        |          ⭐           |        ⭐         |          ⭐           |                |
+| Hyundai    | Ioniq PHEV 2020-21            |    SCC + LKAS     |                        |          ⭐           |        ⭐         |                      |                |
```
Comment (adeebshihadeh) [inline docs/CARS.md]: why not well supported?

File: docs/CARS.md [hunk 6]
```diff
@@ -1,185 +1,200 @@
...
+| Kia        | Niro PHEV 2019                |    SCC + LKAS     |     ⭐<sup>3</sup>      |                      |                  |                      |       ⭐        |
+| Kia        | Optima 2017                   |    SCC + LKAS     |                        |          ⭐           |                  |                      |       ⭐        |
+| Kia        | Optima 2019                   |    SCC + LKAS     |                        |          ⭐           |        ⭐         |                      |       ⭐        |
+| Kia        | Sorento 2018-19               |    SCC + LKAS     |                        |          ⭐           |        ⭐         |                      |                |
+| Kia        | Stinger 2018                  |    SCC + LKAS     |                        |          ⭐           |        ⭐         |                      |       ⭐        |
+| Kia        | Telluride 2020                |    SCC + LKAS     |     ⭐<sup>3</sup>      |          ⭐           |        ⭐         |                      |                |
```
Comment (adeebshihadeh) [inline docs/CARS.md]: same here

**Review by adeebshihadeh — COMMENTED** [review]
Let's make sure we use the same language as on the website.

- [x] define each column, e.g. what does well supported mean? are the gas and brakes still controlled if I don't have "openpilot longitudinal"?
- [x] add a note that cars can move in the tiers

**Comment by khoi** [comment]
The OP experience on my new car HKG Santa Fe 2022 is extremely stressful compares with a Corolla TSS 2, I don't think it deserves to be @ gold tier atm.

**Comment by sshane** [comment]
Closing in favor of https://github.com/commaai/openpilot/pull/23762

**Comment by Verylukyguy** [comment]
You can link my 2018 GMC Acadia video.
https://www.youtube.com/watch?v=0ZN6DdsBUZo

**Pull request closed without merging**